contracts/RealmDiamond/facets/WhitelistFacet.sol

```diff
@@ -0,0 +1,80 @@
+// SPDX-License-Identifier: MIT
```

Comment:
I figured we'd be using the WhitelistFacet from AavegotchiDiamond since it already exists. Is that not possible?

Reply:
@cinnabarhorse You wanna use Whitelists on AavegotchiDiamond?
I thought we can reuse source of WhitelistFacet.

Reply:
If we reuse the AavegotchiDiamond whitelists then people don't need to remake their whitelists. I think it'd be cleaner to do it that way @orionstardust